Gems/PhysX/Code/Tests/EditorShapeColliderComponentTests.cpp

```diff
@@ -161,9 +162,12 @@ namespace PhysXEditorTests
 
         // there should be a single shape on the rigid body and it should be a box
         EXPECT_EQ(pxRigidStatic->getNbShapes(), 1);
-        physx::PxShape* shape = nullptr;
-        pxRigidStatic->getShapes(&shape, 1, 0);
-        EXPECT_EQ(shape->getGeometryType(), physx::PxGeometryType::eBOX);
+        if (pxRigidStatic->getNbShapes() > 0)
```

Comment:
Yes, I had it failing once because the number of shapes returned was 0 and then it caused a crash of the execution because of calling to `shape->getGeometryType()` with shape` being `nullptr`.

Reply:
I thought crashing in a test just meant that test fails and then it continues with the other tests.

Reply:
Maybe in AR it's able to recover, but that was not the case when running from Visual Studio, with makes it annoying that it stops the execution.